dbms/src/Storages/KVStore/MultiRaft/Disagg/CheckpointIngestInfo.h

```diff
@@ -42,6 +42,7 @@ struct CheckpointIngestInfo
     UInt64 regionId() const { return region_id; }
     UInt64 peerId() const { return peer_id; }
     UInt64 beginTime() const { return begin_time; }
+    UInt64 createdTime() const { return begin_time; }
```

Comment:
`return created_time`?

Reply:
And add comments about what is the difference between `beginTime` and `createdTime`

Reply:
Changed this PR back to DNM, because some new functionalities is added into this PR.